### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by neetukumari4858**
Wrong PR
This pull request contains Auth features:
Signup handler
Login handler
Logout handler

### CODE REVIEW

File: src/Routes/AppRouter.js [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
+            <Route path="/VideoListing" element={<VideoListing/>} />
+            <Route path="/LikedPage" element={<RequireAuth> <Liked/></RequireAuth>}/>
+            <Route path="/WatchLater" element={<RequireAuth> <WatchList/></RequireAuth>}/>
+            <Route path="/HistoryPage" element={<RequireAuth> <History /></RequireAuth>}/>
+            <Route path="/PlaylistPage" element={<RequireAuth> <Playlist /></RequireAuth>}/>
+            <Route path="/PlaylistPage/:id" element={<RequireAuth> <SinglePlayList/></RequireAuth>}/>
+            <Route path="/loginPage" element={ <Login/>}/>
+            <Route path="/SignUpPage" element={ <SignUp/>}/>
+            
+            {/* <Route path="/" element= {<Home/>}/>
```
Comment (deepak29-git) [inline src/Routes/AppRouter.js:20]: remove commented code if it is not used

**Review by deepak29-git — APPROVED** [review]
variable name is good

File: src/Routes/AppRouter.js (hunk 1 above)
Comment (neetukumari4858) [reply src/Routes/AppRouter.js:20]: okey I will remove

**Review by Tanjum786 — APPROVED** [review]
code looks clean
nice UI
to me no changes require

**Pull request merged**